### PR TITLE
Set up lerna

### PR DESCRIPTION
**Pull request by Pessimistress**
# For #724 

#### Background
- Maintain one copy of dev dependencies and dev scripts
- Use lerna to manage internal dependencies

#### Change List
- Set up lerna for core and loader.gl modules
- Remove unused scripts

### CODE REVIEW

File: modules/core/package.json [hunk 1]
```diff
@@ -71,25 +71,11 @@
     ]
   },
   "deprecatedScripts": {
-    "build-script": "webpack",
```
Comment (ibgreen) [inline modules/core/package.json]: Just a heads up, this is the `deprecatedScripts` section you are modifying, so changes here won't have any effect. I created it recently when I was restructuring the repo just in case I missed something. Maybe just remove the entire thing to avoid confusion?

File: package.json [hunk 2]
```diff
@@ -26,22 +26,24 @@
     "3d"
   ],
   "workspaces": [
-    "modules/*"
+    "modules/core",
```
Comment (ibgreen) [inline package.json]: So this will no longer yarn install in my other modules, like babel-plugin-... etc?

Why the change? Do both yarn and lerna use this field?

File: package.json [hunk 3]
```diff
@@ -26,22 +26,24 @@
...
-    "build": "(cd modules/core && npm run build)",
+    "build": "lerna run build",
     "cover": "NODE_ENV=cover nyc node test/start",
     "lint": "eslint modules test && npm run lint-yarn",
     "lint-examples": "eslint examples",
     "lint-yarn": "!(grep -q unpm.u yarn.lock) || (echo 'Please rebuild yarn.lock file using public npmrc' && false)",
     "lint-docs": "scripts/lint-markdown.sh",
-    "publish-prod": "(cd modules/core && npm run publish-prod)",
-    "publish-beta": "(cd modules/core && npm run publish-beta)",
+    "publish-prod": "lerna publish --cd-version minor",
```
Comment (ibgreen) [inline package.json]: loaders.gl is currently being broken out to a separate repo. Not sure if it is worth introducing combined publishing here at this stage. But perhaps no harm.

File: package.json (hunk 2 above)
Comment (Pessimistress) [reply package.json]: This is shared by lerna and yarn. The `modules/*` directory needs clean up. Subdirectories without `package.json` or `build` scripts break build.

File: package.json (hunk 3 above)
Comment (Pessimistress) [reply package.json]: I'll do another pass of cleaning up once they are moved.
